intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -814,26 +814,27 @@ \subsection{Generate your own e-mail notifications}
 
 You can instruct the \hpc to send an e-mail to your e-mail address whenever a
 job \strong{b}egins, \strong{e}nds and/or \strong{a}borts, by adding the
-following lines to the job script ``fibo.pbs'':
+following lines to the job script \lstinline|fibo.pbs|:
 
 \begin{code}{bash}
 #PBS -m b
 #PBS -m e
 #PBS -m a
-#PBS -M <your e-mail address>
+#PBS -M john.smith@example.com
 \end{code}
 or
 \begin{code}{bash}
 #PBS -m abe
-#PBS -M <your e-mail address>
+#PBS -M john.smith@example.com
 \end{code}
 
 These options can also be specified on the command line.  Try it and see what
 happens:
 
 \begin{prompt}
-%\shellcmd{qsub -m abe -M <{}your e-mail address>{} fibo.pbs}%
+%\shellcmd{qsub -m abe -M john.smith@example.com fibo.pbs}%
 \end{prompt}
 
-You don't have to specify the e-mail address. The system will
-use the e-mail address which is connected to your VSC account.
+In these examples, we specified the e-mail address as ``john.smith@example.com'',
+but don't have to specify the e-mail address: if you don't specify it, the system will
+use the e-mail address that is connected to your VSC account.
```

Comment:
Turn it around: don't show it in the example, and state that you *can* specify an alternate e-mail address using the `-M` option.